code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -13,8 +13,16 @@
 		var/operating = FALSE
 		var/obj/item/weapon/reagent_containers/beaker = null
 		var/limit = 10
-		var/list/blend_items = list (
+		var/static/list/blend_items
+		var/static/list/juice_items
+		var/static/list/dried_items
 
+		var/list/holdingitems = list()
+
+/obj/machinery/reagentgrinder/Initialize()
+	. = ..()
+	beaker = new /obj/item/weapon/reagent_containers/glass/beaker/large(src)
+	blend_items = typecacheof(list(
```

Comment:
ACtually I'm not sure why I made this a typecache. I think I meant to do just `static`, but since I had my head on typecaches it just happened